dashbuilder/dashbuilder-backend/dashbuilder-external-backend/src/main/java/org/dashbuilder/external/impl/ComponentServiceImpl.java

```diff
@@ -55,11 +56,13 @@
     
     @Override
     public List<ExternalComponent> listAllComponents() {
-        List<ExternalComponent> allComponents = loader.loadExternal();
+        List<ExternalComponent> allComponents = new ArrayList<>();
+        List<ExternalComponent> external = loader.loadExternal();
```

Comment:
The list `external` seems to be useless here. Am I missing something?

Reply:
You are right, I forgot this line, added in my latest commit and enabled the test that checks it (strangely the test didn't fail - the reason was it was not enabled!!)

Reply:
Great! Now we are safe :)